addons/packages/fluxcd-source-controller/metadata.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: data.packaging.carvel.dev/v1alpha1
+kind: PackageMetadata
+metadata:
+  name: fluxcd.source.controller.community.tanzu.vmware.com
+spec:
+  displayName: "Flux Source Controller"
+  longDescription: "The source-controller is a Kubernetes operator, specialised in artifacts acquisition from external sources such as Git, Helm repositories and S3 buckets."
+  shortDescription: "The source-controller is a Kubernetes operator, specialised in artifacts acquisition from external sources such as Git, Helm repositories and S3 buckets."
+  supportDescription: "https://tanzu.vmware.com/support"
+  providerName: "VMware"
+  categories:
+  - policy
```

Comment:
should we update the category here? for the other fluxcd packages we are using gitops.

Reply:
Is this as simple as switching from policy to gitops in the categories section?

Reply:
Changed!